app/src/main/java/com/hedvig/app/feature/insurance/ui/detail/coverage/CoveredAndExceptionModel.kt

```diff
@@ -0,0 +1,19 @@
+package com.hedvig.app.feature.insurance.ui.detail.coverage
+
+sealed class CoveredAndExceptionModel {
+    data class Icon(val link: String) : CoveredAndExceptionModel()
+    data class Title(val text: String) : CoveredAndExceptionModel()
+    data class Description(val text: String) : CoveredAndExceptionModel()
+    sealed class CommonDenominator : CoveredAndExceptionModel() {
```

Comment:
`CommonDenominator` was an example name 😬 

If these are as similar as the code suggested, we should be able to find a common ancestor name - for example, the Headers have `CoveredAndExceptionModel.>>Header<<.` where `Header` is the 'common denominator' in this case.

Reply:
🤦